seqz-preprocess/main.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
this main.py file is generated at the time the package was created, it is indeed completely optional. If not used, it can be removed.

Reply:
thanks I will remove it